cli/beamable.common/Runtime/JsonSerializable/LoadStream.cs

```diff
@@ -70,6 +71,15 @@ public void DeregisterISerializableFactory(ISerializableFactory factory)
 				factories.Remove(factory);
 			}
 
+			public bool SerializeNestedJson(string key, ref JsonString jsonString)
+			{
+				if (curDict.TryGetValue(key, out var tmp))
+				{
+					jsonString = JsonString.FromValue(tmp);
+				}
+				return true;

```

Comment:
Same question as above here, what is the purpose of the bool return if it will always be the same?

Reply:
Same idea here --- when deserializing, if we have some value in the JSON blob, we create a `JsonString` from it --- otherwise, we don't create the `JsonString` instance.